lean4-infoview/src/infoview/goals.tsx

```diff
@@ -158,13 +158,15 @@ export const Goal = React.memo((props: GoalProps) => {
 interface GoalsProps {
     goals: InteractiveGoals
     filter: GoalFilterState
+    displayCount: boolean
 }
 
-function Goals({ goals, filter }: GoalsProps) {
+function Goals({ goals, filter, displayCount}: GoalsProps) {
     if (goals.goals.length === 0) {
         return <>No goals</>
     } else {
         return <>
+            {displayCount && <strong className="goal-goals">{goals.goals.length} goals</strong>}
```

Comment:
```suggestion
    const nGoals = goals.goals.length
    if (nGoals === 0) {
        return <>No goals</>
    } else {
        return <>
            {displayCount &&
                <strong className="goal-goals">{nGoals} {1 < nGoals ? 'goals' : 'goal'}</strong>}

```
(If we keep this variant.)